packages/plugins/flow/resolvers/tests/flow-resolvers.spec.ts

```diff
@@ -192,9 +194,7 @@ describe('Flow Resolvers Plugin', () => {
       { outputFile: '' }
     ) as Types.ComplexPluginOutput;
 
-    expect(result.prepend).not.toContain(
-      `import { type GraphQLResolveInfo, type GraphQLScalarTypeConfig } from 'graphql';`
-    );
+    expect(result.prepend).toContain(`import { type GraphQLResolveInfo } from 'graphql';`);
```

Comment:
The test name and assertion never matched before, so it's no wonder this never worked and the tests passed.